src/screens/ExchangeModal.js

```diff
@@ -68,19 +68,22 @@ const Wrapper = ios ? KeyboardFixedOpenLayout : Fragment;
 
 const InnerWrapper = styled(Centered).attrs({
   direction: 'column',
-})`
-  ${ios
-    ? position.sizeAsObject('100%')
-    : `
-    height: 500;
-    top: 0;
-  `};
-  background-color: ${({ theme: { colors } }) => colors.transparent};
-`;
-
-const Spacer = styled.View`
-  height: 20;
-`;
+})(
+  ios
+    ? {
+        ...position.sizeAsObject('100%'),
+        backgroundColor: ({ theme: { colors } }) => colors.transparent,
```

Comment:
This property is duplicated across both states so should be hoisted out of the ternary.